Dockerfile.web

```diff
@@ -1,6 +1,6 @@
 ARG image_tag=latest
-FROM elifesciences/journal_assets:${image_tag} AS assets
+FROM elifesciences/journal:${image_tag} AS app
 FROM nginx:1.13.12-alpine
 
 COPY .docker/nginx-default.conf /etc/nginx/conf.d/default.conf
-COPY --from=assets /web/ /srv/journal/web/
+COPY --from=app /srv/journal/web/ /srv/journal/web/
```

Comment:
Will still need to push this image separately from the main `app` one

Reply:
Wasn’t anticipating pushing an Nginx image (no need for the assets to go into `app` if we do).

Reply:
The `web` image contains the assets now copied to `/srv/journal/web/` of the assets, without pushing it would nginx serve them? Forwarding the requests to fpm, mounting some kind of shared volume, producing one image on the fly by copying this folder?

Reply:
In docker-compose, there is the [usual trick](https://stackoverflow.com/questions/42232051/docker-compose-volumes-from-equivalent-with-version-3#42234691
) of mounting a volume, but the assets then outlive the containers making it more amenable to have version mismatches rather than less. Producing a new image requires a `docker-compose build` for every new node; pushing it from the build phase would not include the configuration but only the assets as they are the part that is identical across environments.

For the long-run, Kubernetes support is also nebulous as it may share data between containers in the same pod (colocated on the same host) with a [empty dir volume]( https://kubernetes.io/docs/concepts/storage/volumes/) but it requires the files to be copied manually into that volume with a separate InitContainer. Unclear how long is the lifecycle of that volume.